controller/space_test.go

```diff
@@ -395,20 +395,20 @@ func (rest *TestSpaceREST) TestShowSpaceNotModifiedUsingIfNoneMatchHeader() {
 	assert.Equal(t, *created.Data.Attributes.Version, *fetched.Data.Attributes.Version)
 
 	// verify list-WI URL exists in Relationships.Links
-	require.NotNil(t, *fetched.Data.Relationships.Workitems)
-	require.NotNil(t, *fetched.Data.Relationships.Workitems.Links)
-	require.NotNil(t, *fetched.Data.Relationships.Workitems.Links.Related)
+	require.NotNil(t, fetched.Data.Relationships.Workitems)
```

Comment:
Yes yes @kwk this totally makes sense.
In case we need to check value is not-nil then 2 checks -
`require.NotNil(t, fetched.Data.Relationships.Workitems.Links)`
`require.NotNil(t, *fetched.Data.Relationships.Workitems.Links)`